### DISCUSSION TITLE
i want to add more words for my language

### DISCUSSION THREAD

**Discussion by aviegdown** [monkeytypegame/monkeytype#756] [Ideas] [answered]
based on  [static/languages/indonesian.json](https://github.com/Miodec/monkeytype/blob/master/static/languages/indonesian.json) currently my language is only have 141 words, i wanna add more or maybe create indonesian 1k. can i do that? thanks

**Comment by Miodec** [comment, marked as answer]
You could start off by making indonesian have 200 words, and them move onto creating a 1k version - go for it!
